docs/configuration/profile.md

```diff
@@ -32,6 +32,56 @@ Add a packer build
 * `source_ami` The source AMI ID for an `amazon-ebs`
 * `ssh_username` Default `ubuntu`
 * `ami_virtualization_type` Default `hvm`
+* `tagging_role` the name of an IAM role that exists in each remote account that allows the AMI to be retagged
+
+  Example usage:
+
+  <pre>
```

Comment:
My problem is that I can never get the indentation right when I'm already inside an indented block